tests/data/dsl-semantic-validation-cases.yaml

```diff
@@ -1184,15 +1184,6 @@
         end: 1
       metadata:
         errorType: schema-version-required
-    - msg: invalid schema undefined
```

Comment:
The error message here was proving difficult to align between JS and Java, and then I kinda realised well what's the point in `invalid schema undefined` if we're already saying `there is no schema version`. They seem like mutually exclusive errors imo

@d-jeffery @rhamzeh wdyt?